client/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "client",
   "version": "0.1.0",
   "private": true,
+  "proxy": "http://localhost:4000",
```

Comment:
This is just used for development, you can check in here:
https://create-react-app.dev/docs/proxying-api-requests-in-development

The port :4000 is the port i've used for my server locally. In your cases it might be different